### PR TITLE
fix(x/ecocredit): register ecocredit v1alpha1 messages

### PR DESCRIPTION
**Pull request by ryanchristo**
## Description

Closes: #1356

This pull request registers ecocredit v1alpha1 messages to allow for historical events to be queried.

The proto break check is failing although this is not a proto breaking change, only the file option was updated. Implementing the messages any further does not seem necessary given we do not register the messages to the msg server. This should qualify for a point release, no API or state machine breaking changes, but let me know if there are any concerns.

Manual Test: https://hackmd.io/USRbzfq1QwSn2oJcXwQ3Xg

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by technicallyty — APPROVED** [review]
LGTM

File: x/ecocredit/migrations/v3/types.pb.go [hunk 1]
```diff
@@ -466,11 +466,11 @@ func (m *CreditTypeSeq) GetSeqNumber() uint64 {
 }
 
 func init() {
-	proto.RegisterType((*ClassInfo)(nil), "regen.ecocredit.v1alpha1.ClassInfo")
-	proto.RegisterType((*BatchInfo)(nil), "regen.ecocredit.v1alpha1.BatchInfo")
-	proto.RegisterType((*Params)(nil), "regen.ecocredit.v1alpha1.Params")
-	proto.RegisterType((*CreditType)(nil), "regen.ecocredit.v1alpha1.CreditType")
-	proto.RegisterType((*CreditTypeSeq)(nil), "regen.ecocredit.v1alpha1.CreditTypeSeq")
+	//proto.RegisterType((*ClassInfo)(nil), "regen.ecocredit.v1alpha1.ClassInfo")
```
Comment (aleem1314) [inline x/ecocredit/migrations/v3/types.pb.go:469]: Why these lines commented?

**Review by aleem1314 — APPROVED** [review]
LGTM

File: x/ecocredit/core/v1alpha1/types.pb.go [hunk 2]
```diff
@@ -0,0 +1,2170 @@
...
+	}
+	return 0
+}
+
+func init() {
+	proto.RegisterType((*ClassInfo)(nil), "regen.ecocredit.v1alpha1.ClassInfo")
+	proto.RegisterType((*BatchInfo)(nil), "regen.ecocredit.v1alpha1.BatchInfo")
+	proto.RegisterType((*Params)(nil), "regen.ecocredit.v1alpha1.Params")
+	proto.RegisterType((*CreditType)(nil), "regen.ecocredit.v1alpha1.CreditType")
+	proto.RegisterType((*CreditTypeSeq)(nil), "regen.ecocredit.v1alpha1.CreditTypeSeq")
```
Comment (ryanchristo) [inline x/ecocredit/core/v1alpha1/types.pb.go:473]: Answering the above question, the types are registered here.

File: x/ecocredit/migrations/v3/types.pb.go (hunk 1 above)
Comment (ryanchristo) [reply x/ecocredit/migrations/v3/types.pb.go:469]: We register the types here: https://github.com/regen-network/regen-ledger/pull/1360#discussion_r949333351

**Comment by ryanchristo** [comment]
Lint passed here but failed on master :thinking: 

https://github.com/regen-network/regen-ledger/runs/7905079611?check_suite_focus=true

**Pull request merged**